package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-titanium",
   "description": "grunt plugin for titanium CLI",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
@FokkeZB bad open source etiquette, really shouldn't change version numbers in PRs. Remove that please.